aiida/cmdline/commands/cmd_process.py

```diff
@@ -324,3 +344,22 @@ def _print(communicator, body, sender, subject, correlation_id):  # pylint: disa
 
         # Reraise to trigger clicks builtin abort sequence
         raise
+
+
+@verdi_process.command('revive')
+@decorators.only_if_daemon_not_running()
+@click.pass_context
+def cmd_revive(ctx):
+    """Revive all stuck processes.
+
+    N.B.: This command requires the daemon to be stopped.
+
+    This command will perform a query of the database to find all "active" processes, meaning those that haven't yet
+    reached a terminal state, and cross-references this with the active process tasks that are in the process queue of
+    RabbitMQ. Any active process that does not have a corresponding process task can be considered a zombie, as it will
+    never be picked up by a daemon worker to complete it and will effectively be "stuck". Any process task that does not
+    correspond to an active process is useless and should be discarded. Finally, duplicate process tasks are also
+    problematic and are discarded.
```

Comment:
```suggestion
    This command queries the database to find all "active" processes, meaning those that haven't yet
    reached a terminal state, and cross-references them with the active process tasks in the process queue of
    RabbitMQ. Any active process that does not have a corresponding process task can be considered a zombie, as it will
    never be picked up by a daemon worker to complete it and will effectively be "stuck". Any process task that does not
    correspond to an active process is useless and should be discarded. Finally, duplicate process tasks are also
    problematic and are discarded.
```

Reply:
One thing that isn't clear from this doc: does it do anything about the zombie?